splunk-otel-android/src/main/java/com/splunk/rum/Config.java

```diff
@@ -38,6 +38,7 @@
 
     public static final boolean DEFAULT_ENABLE_SLOW_RENDERING_DETECTION = true;
     public static final Duration DEFAULT_SLOW_RENDER_POLLING_INTERVAL = Duration.ofSeconds(1);
+    public static final double DEFAULT_MAX_UNCOMPRESSED_BANDWIDTH = 15.0 * 1024;
```

Comment:
That's not configurable right now; WDYT about moving it to the `DiskToZipkinExporter` class, where it's actually used?